blocks/back-button.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+
+export function BackButton() {
```

Comment:
Move this component in `components` folders. In blocks we keep only big sections